pkg/broker/filter/filter_handler.go

```diff
@@ -81,12 +82,16 @@ func NewHandler(logger *zap.Logger, triggerInformer v1.TriggerInformer, reporter
 		MaxIdleConnsPerHost: defaultMaxIdleConnectionsPerHost,
 	})
 
+	fm := subscriptionsapi.NewFiltersMap()
+
 	triggerInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
 			trigger, ok := obj.(eventingv1.Trigger)
 			if !ok {
 				return
 			}
+			logger.Debug("Adding filter to filtersMap")
+			fm.Set(fmt.Sprintf("%s.%s", trigger.Namespace, trigger.Name), createSubscriptionsAPIFilters(logger, trigger.Spec.Filters))
```

Comment:
Can we extract a function to perform the key given the trigger?

Reply:
Alternatively we could pass the trigger the logic is encapsulated inside `FiltersMap`

Reply:
I think passing the trigger makes sense